File: cs/proxy/db/SiginOperation.cs
/// <summary>
/// 签到
/// </summary>
namespace org.zgl.service.client.db{
	[Proxy(gameId = -7)]
	public interface SiginOperation : ITcpAsyncService { 
		/// <summary>
		/// 签到奖励
		/// </summary>
		void sigin(long arg0);
		/// <summary>
		/// 签到奖励 ----> <<sigin();回调>>
		/// </summary>
		void sigin2CallBack(ItemListDto callBackParam);
		/// <summary>
		/// 签到转盘
		/// </summary>
		void dial(long arg0);
		/// <summary>
		/// 签到转盘 ----> <<dial();回调>>
		/// </summary>
		void dial2CallBack(ItemListDto callBackParam);
	}
}

File: componet/src/main/java/org/zgl/service/client/db/SiginOperation.java
package org.zgl.service.client.db;

import org.zgl.dto.ItemListDto;
import org.zgl.proxy.ClassDesc;
import org.zgl.proxy.MethodDesc;
import org.zgl.proxy.rule.ITcpAsyncService;

/**
 * @author： 猪哥亮
 * @创建时间： 2018/8/17
 * @文件描述：
 */
@ClassDesc("签到")
public interface SiginOperation extends ITcpAsyncService {
    @MethodDesc("签到奖励")
    ItemListDto sigin(long uid);
    @MethodDesc("签到转盘")
    ItemListDto dial(long uid);
}
